mlp/training/mlp_model.py

```diff
@@ -50,38 +50,75 @@ def generate_default_model(self, num_labels):
         bias_init = Constant(val=1.0)
 
         layers = []
+    
         layers.append(Affine(
-            nout=500,
+            nout=250,
             init=init_norm,
             bias=bias_init,
             activation=Rectlin()))
 
         layers.append(Dropout(
-            name="do_1",
+            name="do_2",
             keep=0.9))
-    
+
         layers.append(Affine(
-            nout=250,
+            nout=100,
             init=init_norm,
             bias=bias_init,
             activation=Rectlin()))
 
         layers.append(Dropout(
-            name="do_2",
+            name="do_3",
             keep=0.9))
 
         layers.append(Affine(
+            nout=num_labels,
+            init=init_norm,
+            bias=bias_init,
+            activation=Logistic()))
+
+        model = Model(layers=layers)
+        return model
+
+    def generate_slacking_model(self, num_labels):
```

Comment:
How about creating an extra class for the models? They are static anyway and don't really belong to the trainer.